infra/test/integration/reviewsite/serverspec/default_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'serverspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.